factory/dataComponents_test.go

```diff
@@ -141,80 +141,12 @@ func getDataArgs() factory.DataComponentsFactoryArgs {
 
 func getGeneralConfig() *config.Config {
 	return &config.Config{
```

Comment:
this function it looks ugly.
Can you create functions for each of the component configurations that can't be written in less than 3 lines?
e.g storagePruningCfg, storageCfg(filepath) - you can use this for all storages with common configuration, etc.

Reply:
you're right.. also, no need for the path to be specified for these tests. refactored